examples/applications/plot_out_of_core_classification.py

```diff
@@ -362,7 +366,10 @@ def plot_accuracy(x, y, x_legend):
 
 cls_runtime.append(total_vect_time)
 cls_names.append('Vectorization')
-bar_colors = rcParams['axes.color_cycle'][:len(cls_names)]
+if ver<'1.5.1':
```

Comment:
though it works until single digit major version, it does lexicographic comparison. It fails for double digit numbers